vm/devices/get/guest_emulation_transport/src/client.rs

```diff
@@ -465,6 +465,12 @@ impl GuestEmulationTransportClient {
         self.control.call(msg::Msg::FlushWrites, ()).await
     }
 
+    /// Report the event to host and flush the event queue.
+    pub async fn event_log_and_flush(&self, event_log_id: crate::api::EventLogId) {
```

Comment:
I would call this API `event_log_fatal`, as really, that's the intent behind this pattern - making sure events related to fatal errors are suitably flushed prior to tearing down the rest of OpenHCL. It also makes it immediately clear to both existing and future callers of this API when to use this single-function variant, vs. the existing split-function variants.

Reply:
updated